setup.py

```diff
@@ -0,0 +1,57 @@
+from setuptools import setup
+import re
+
+__version__ = re.findall(
+    r"""__version__ = ["']+([0-9\.]*)["']+""",
+    open("pyxlight/__init__.py").read(),
+)[0]
+
+setup(
+    name="pyXLIGHT",
+    version=__version__,
+    description="A Python wrapped version of Mark Drela's XFOIL code with the GUI features removed.",
+    long_description="""
```

Comment:
This can be replaced by reading in the README.md file, see [here](https://github.com/mdolab/adflow/blob/master/setup.py#L10-L12) for an example

Reply:
Updated.